### PR TITLE
Activate Jobs using Java gRPC client

### LINKED ISSUE #1426: I can activate a batch of jobs using Java gRPC client
The gRPC protocol exposes a method to activate a batch of jobs. The message contains the request has the properties:
- String type
- String worker;
- long timeout (in ms)
- int amount;

The response contains a list of jobs with the properties:
- key
- type
- headers
- custom headers
- worker
- retries
- deadline
- payload

### PR DESCRIPTION
**Pull request by menski**
- adds new ActivateJobs gRPC method
- adds newActivateJobsCommand to JobClient to create new command
- gateway will request jobs from all partitions in round robin fashion

closes #1426

### CODE REVIEW

File: clients/java/src/main/java/io/zeebe/client/api/commands/ActivateJobsCommandStep1.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     * @param timeout the time as duration (e.g. "Duration.ofMinutes(5)")
+     * @return the builder for this command. Call {@link #send()} to complete the command and send
+     *     it to the broker.
+     */
+    ActivateJobsCommandStep3 timeout(Duration timeout);
+
+    /**
+     * Set the name of the subscription owner.
+     *
+     * <p>This name is used to identify the worker to which a job is exclusively assigned to.
```
Comment (npepinpe) [inline clients/java/src/main/java/io/zeebe/client/api/commands/ActivateJobsCommandStep1.java]: Jobs aren't exclusively assigned to a worker though; it's just harder to figure out which jobs are activated at a time. I mean, it's an extremely minor point, I'm just wondering if we ever plan on clarifying these things to users.

**Review by npepinpe — CHANGES_REQUESTED** [review]
Looks good in general.

One thing missing is gateway-broker tests (or at least more complete IT tests), and the retries property for the ActivateJobs request. We used to set retries on subscriptions, but I didn't see where this is done now. Wouldn't it make sense on the request?

**Comment by menski** [comment]
@npepinpe should be ready for review again. Please see the commits from bfc2dea onwords. I will squash all the commits in the actual feature commit after review.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d6e6565beada67d6462dbc9687d8af5e5f454399
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Review by npepinpe — APPROVED** [review]
Looks good, I'll let you rewrite the history and merge.

**Comment by menski** [comment]
bors retry

**Pull request merged**